src/main/java/com/turkcell/staj/dtos/transaction/requests/RequestUpdateTransactionDTO.java

```diff
@@ -0,0 +1,38 @@
+package com.turkcell.staj.dtos.transaction.requests;
+
+import com.turkcell.staj.core.enums.Status;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.PositiveOrZero;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDate;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class RequestUpdateTransactionDTO {
+    @NotNull(message = "Transaction can't be null")
+    @Min(value = 1, message = "Id must be a positive integer greater than or equal to 1")
+    private int transactionId;
```

Comment:
Id update edilmemeli

Reply:
parametre olarak gelen id'yi object'e kaydetmek için o alan var. son kullanıcının hiç bir etkisi yok.

Reply:
Tamamdır bu konuyu tekrar konuşalım